test/core/xds/xds_transport_fake.h

```diff
@@ -117,6 +127,9 @@ class FakeXdsTransportFactory : public XdsTransportFactory {
     std::deque<std::string> from_client_messages_ ABSL_GUARDED_BY(&mu_);
     bool status_sent_ ABSL_GUARDED_BY(&mu_) = false;
     bool orphaned_ ABSL_GUARDED_BY(&mu_) = false;
+    size_t read_count_ ABSL_GUARDED_BY(&mu_) = 0;
```

Comment:
Suggest calling this `reads_started_`, to make it clear that we're counting reads started instead of reads completed.